sign_up_window

p0u74 opened the pull request:
Changes applied :

- Class name changed to SignUpWindow
- constants.py created
- Window icon location added as constant
- Imported constants

p0u74 committed:
fix: class name changed to SignUpWindow

p0u74 committed:
constants.py added

p0u74 committed:
fix: constants imported

window icon location added as constants

p0u74 committed:
fix: deleted additional imports

alirzaMhd commented on sign_up_window.py:
delete useless comments (all of yours are useless)
rewrite this in the **DRY** principle way
I told you to read and write this module based on **MVC** did you

<html>
<body>
<!--StartFragment-->

Type of Comments | Good | Bad | Example
-- | -- | -- | --
Autogenerated Comments | None | Redundant | N/A
Clarification Comments | Rarely | Often | // Increment the counter.
Commented-Out Code | None | Creates Dead Code | // someFunction();
Marker Comments | Sometimes | Unclear Intent | // BEGIN MAIN PROGRAM
TODO Comments | Rarely | Often Left Unresolved | // TODO: Refactor this function
Amplification Comments | Sometimes | Non-essential Information | // This function calculates the sum of two numbers.
Javadocs, XmlDocs, etc. | Sometimes | Errors, Incorrect Information | /** This method returns the name of the person. **/
Commented-Out Tests | None | Creates False Sense of Security | N/A
Legal Comments | Sometimes | Unmaintained, Misleading | // Copyright (c) 2021 Company Name
Explanation of Intent | Sometimes | Misleading, Duplication | // Check if the input is valid.
Explanation of Implementation | Seldom | Error-Prone | // Use quicksort algorithm to sort the array.
Warning of Consequences | Sometimes | Conflicts with Code | // Be careful while modifying this code.
TODO Comments (Implementation) | Seldom | Code Should be Well-Designed | // TODO: Implement this function using a better algorithm.
Debugging Comments | Seldom | Debuggers are More Effective | // Print debug information.
Global Information | Seldom | Obvious, Distracting | // This is the main function.

<!--EndFragment-->
</body>
</html>


**if you had any more question --> [google](googl.com)**

p0u74 committed:
fix: comments removed

p0u74 committed:
fix: comments removed

p0u74 committed:
fix: create_widgets added

p0u74 committed:
fix: setGeometry deleted

p0u74 committed:
fix: book_search added

alirzaMhd commented on book_search/book_search.py:
```diff
@@ -0,0 +1,28 @@
+        window_layout.addWidget(search_bar)
+
+        search_btn = QPushButton("Search", self)
+        window_layout.addWidget(search_btn)
```
so you've learned about DRY right?
I totally remember to mention this line and line 23 does the same thing!!!!!!!!!!!!!!!

alirzaMhd commented on sign_up_window.py:
```diff
@@ -1,27 +1,26 @@
+        
         input_box = QLineEdit(self)
-        input_box.setGeometry(150, 400, 200, 30)
         window_layout.addWidget(input_box)
```
as in here .... DRY please

alirzaMhd committed:
Delete book_search.py

this file should be in its branch WTF?
sign_up_window branch and book_search?????????????

alirzaMhd commented on book_search/constants.py:
```diff
@@ -0,0 +1 @@
+WINDOW_ICON = "icon/app_icon.ico"
```
wtf is this?

alirzaMhd commented on book_search/main.py:
```diff
@@ -0,0 +1,12 @@
+import sys
+
```
and wtf is this??????
